### DISCUSSION TITLE
Whether there are plans to release firmware that supports Ethernet and external memory(PSRAM)

### DISCUSSION THREAD

**Discussion by jingqunhu** [nanoframework/Home#1468] [General]
An attempt to compile the firmware failed. 
Now only ESP32_LILYGO,ESP32_OLIMEX, but they can not support external memory, in the process of use, if the communication data is too much, there will be insufficient memory problems.
So ask if our organization will consider this firmware.

**Comment by josesimoes** [comment]
You can adjust the SDK config accordingly and build locally. That's the quickest way to have such firmware image.
This is made to be as flexible as possible. It wouldn't be feasible for us to build and distribute all feature combinations.

**Reply by jingqunhu** [reply]
Okay, I see. If I'm willing to support some of the funds to do this, I don't know if it's possible to consider this. My email listenrainy@126.com, look forward to further communication

**Reply by jingqunhu** [reply]
All I need is a firmware with large memory and network support. because I didn't have time to understand and compile firmware with network.

**Reply by josesimoes** [reply]
We've discussed this internally and this is what is going to happen:
- Fw images ETH capable will be listed in a separate section to improve discovery.
- New variants will be added to cover more devices, including the one you're looking for.

Is not possible to commit to an availability date at this point, but I can tell you it will be done soon.
In the meantime, funding the project is very much appreciated. 
Please use [GitHub Sponsors](https://github.com/sponsors) or [Open Collective](https://opencollective.com/nanoframework).

**Reply by jingqunhu** [reply]
I'm really happy to hear this. I hope this version comes out sooner. And I hope the organization will get more and more fire
